shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -170,7 +170,15 @@ private void copyPath(File src, AlluxioURI dstPath) throws IOException {
           closer.close();
         }
       } else {
-        mFileSystem.createDirectory(dstPath);
+        // Add a judgement, because the dstPath maybe already exists. For example:
+        // 'copyFromLocal pathToSrcDir /'.
+        try {
+          mFileSystem.createDirectory(dstPath);
+        } catch (FileAlreadyExistsException e) {
```

Comment:
By design, `copyFromLocal` works differently like unix `cp`, when you want to copy srcDir under dstDir, you need to say `copyFromLocal srcDir dstDir/srcDir`.